package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "cryptowatch-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
+    "big.js": "^6.2.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.11.0",
+    "react-scripts": "5.0.1",
+    "uid": "^2.0.2",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
The issue with the failing test is that your package.json file does not have `react-test-renderer` listed as a dependency. This causes your tests to fail when your project is cloned to another machine, as the module is not found after installing all the dependencies and running tests. Please update your package.json file to include `react-test-renderer` as a dependency. Also, check to make sure that this update has been pushed to GitHub

<img width="301" alt="image" src="https://github.com/bilalrajput09/CryptoWatch-React-App/assets/37116322/e98d7eba-8dd8-4291-8a3c-a11585bcafdb">

Reply:
Alright. Fixed ✔